addons/binding/org.openhab.binding.evohome/README.md

```diff
@@ -0,0 +1,122 @@
+|-------------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Temperature       | Number    | Allows for viewing the current actual temperature of the zone.                                                                                                                                                         |
+| CurrentSetPoint   | Number    | Allows for viewing the current temperature set point of the zone.                                                                                                                                                      |
+| SetPointStatus    | String    | Allows for viewing the current set point mode of the zone.                                                                                                                                                             |
+| PermanentSetPoint | Number    | Allows for permanently overriding the temperature set point of the zone.                                                                                                                                               |
+| CancelSetPoint    | Switch    | Allowing for cancelling any active temperature overrides on the zone and allowing it to follow the active schedule. Activate by sending `ON` to the channel; it will auto-reset to `OFF`. Sending `OFF` has no effect. |
+
+## Full Example
+
+### demo.things
+
+    Bridge evohome:account:your_account_alias [ username="your_user_name", password="your_password" ]
+    {
+        display your_display_alias  [ id="1234" ]
+        heatingzone your_zone_alias [ id="5678" ]
+    }
+    
+    
+### demo.items  
+
+    // evohome Display
+    String DemoMode                 { channel="evohome:display:your_account_alias:your_display_alias:SystemMode" }
+
+    // evohome Heatingzone
+    Number DemoZoneTemperature      { channel="evohome:heatingzone:your_account_alias:your_zone_alias:Temperature" }
+    Number DemoZoneSetPointValue    { channel="evohome:heatingzone:your_account_alias:your_zone_alias:CurrentSetPoint" }
+    String DemoZoneSetPointStatus   { channel="evohome:heatingzone:your_account_alias:your_zone_alias:SetPointStatus" }
+    Number DemoZoneSetPointOverride { channel="evohome:heatingzone:your_account_alias:your_zone_alias:PermanentSetPoint" }
+    Switch DemoZoneSetPointCancel   { channel="evohome:heatingzone:your_account_alias:your_zone_alias:CancelSetPoint" }
+    
+### demo.sitemap
+    
+    sitemap evohome label="evohome Menu"
+    {
+        Frame label="evohome display" {
+            Selection label="[%s]" item=DemoMode mappings=[
```

Comment:
The mappings should no longer be necessary, the UIs do that automatically.

Reply:
Mine doesn't :-) Leave it there for completeness sake?